### ISSUE TITLE
Bracket Pair Colorization: Getting Started View

### ISSUE THREAD

**Issue by hediet** [microsoft/vscode#130557]
Once https://github.com/microsoft/vscode/pull/129231 lands, I think it would be nice to mention it in the Getting Started view.

Bracket Pair Colorization will be disabled by default and requires users to explicitly enable it, so I think many users might discover this functionality in the getting started view if it is mentioned there.

**Comment by heartacker** [comment]
I believe that we should enable it by default, at the same time, also  ask users to make a discission whether to disable those extensions which have same feature like BPC/2, or maintain the present status

**Comment by misolori** [comment]
Any thoughts about just enabling by default in Insiders at least? We could always turn it off before we ship if we aren't confident in it yet. We do that a lot with new features for increased awareness.

**Comment by JacksonKearl** [comment]
A category dedicated to "These are some settings that we think you might want to tweak" would be nice to have, this would be a good contribution there.

Agree that enabling it by default on insiders makes good sense

**Comment by misolori** [comment]
This would fit nicely under the "Boost your productivity" walkthrough

**Comment by hediet** [comment]
Now that the feature is available in the Insider build, I think it makes sense to tackle this issue.
What do I need to do for this?

**Comment by JacksonKearl** [comment]
You can just add any entry here:
https://github.com/microsoft/vscode/blob/0214d59846f265d9e50f53b61e2267407d81cbfb/src/vs/workbench/contrib/welcome/gettingStarted/common/gettingStartedContent.ts#L284

You'll want to have a themed SVG to go with it, and example is here: https://github.com/microsoft/vscode/blob/920303b974e9ac5b9f7d30d5bc354db5a849c793/src/vs/workbench/contrib/welcome/gettingStarted/common/media/settingsSync.svg#L4

You can either make that yourself in {vector image editor of your choice} or ask someone on the design team nicely 😉

Also FYI the `bracket-pair-colorization` label should be colored `c5def5`, both to visually go with the other sub-feature-areas and because some tooling (for instance the triage bot) use colors as a tagging mechanism.

**Comment by hediet** [comment]
I'm not 100% sure where to mention bracket pair colorization options.

Shall I just add another option (e.g. "Enable Bracket Pair Colorization") here:

![image](https://user-images.githubusercontent.com/2931520/130443015-60b602a2-efff-4e7d-a5c1-094c8a468416.png)

?

However, @isidorn had the interesting idea of adding an entire section for editor related stuff (which also includes multi-cursor stuff)

**Comment by isidorn** [comment]
Two more things I mentioned in our chat (just to have it here)

1. Think about adding it to the `Commonly Used` section in the Settings UI fyi @roblourens 
2. Add it to the tips-and-tricks section of our docs https://code.visualstudio.com/docs/getstarted/tips-and-tricks

Now that I think about it a bit more I am not 100% certain this should be in the Getting Started, because of course a user can get started without this feature. It is just a nice feature on top. So depends on how much we want to have "Tips and Tricks" in the "Getting Started" and how much we want it to be simple flow such that new users can just start coding.

My point is: we should market this cool feature, and maybe we should all the other things (twitter, tiktok, docs, blog) except getting started.

**Comment by hediet** [comment]
I'm open for not including it in this welcome screen.

**Comment by JacksonKearl** [comment]
Going to re-open with (new) tips and tricks label for tracking if we do add a dedicated "tips and tricks" section in the future (which I'd like to do).

**Comment by misolori** [comment]
Would be nice if we had a "tips & tricks" section that appeared after some time (like let's say 3 weeks) so that we don't overwhelm the walkthrough content as they start to accumulate.

**Comment by JacksonKearl** [comment]
This is enabled by default now so does not need the entry.
